lib/block-editor-settings.php

```diff
@@ -78,6 +78,7 @@ function gutenberg_get_block_editor_settings( $settings ) {
 
 	$settings['styles'] = array_merge( $global_styles, get_block_editor_theme_styles() );
 
+	$settings['__experimentalStyles']   = gutenberg_get_global_styles();
```

Comment:
Thanks, I have spotted that one but only had a chance to glance at it quickly. It seems pretty involved for just exposing the global styles data in the block editor. 

However it shakes out, I can easily update this PR to retrieve the data from whatever eventually gets exposed.

Reply:
It's unusual that there's `__experimentalStyles` in addition to `styles`. Technically, `styles` should be the stabilized version of `__experimentalStyles`, but they're coexisting. 😬 

I guess not introduced here, but might be worth making a separate issue for the tech debt.

Reply:
Agreed.

As you noted, the naming wasn't introduced in this PR. I'd be up for a complete rename that maybe reflects that this is the style data as opposed to the compiled CSS styles. 

It's because only the compiled styles were added to the store previously that this change is needed.

Reply:
AFAIK __experimentalStyles is only on mobile now, so let's avoid introducing it to web. We should probably rely on https://github.com/WordPress/gutenberg/pull/59929 and the Rest API here instead of adding a new setting from backend to frontend.